pypistats/__init__.py

```diff
@@ -15,6 +15,8 @@
 
 BASE_URL = "https://pypistats.org/api/"
 
+FORMATS = ("json", "markdown")  # only used for printing to terminal
```

Comment:
Would this be better kept internal to `cli.py`?

Reply:
I'll do it.